apps/www/_blog/2022-08-12-supabase-series-b.mdx

```diff
@@ -0,0 +1,181 @@
+---
+title: 'Supabase Series B'
+description: Supabase just raised $80M, bringing our total funding to $116M.
+author: paul_copplestone
+image: series-b/supabase-series-b.png
+thumb: series-b/supabase-series-b.png
+tags:
+  - supabase
+date: '2022-08-12'
+toc_depth: 3
+---
+
+Supabase raised $80M in May, bringing our total funding to $116M. This round was around one year after our Series A, and we figured we'd revisit the plans
+we outlined in our [Series A blog post](https://supabase.com/blog/supabase-series-a) and hold ourselves accountable to some of the promises we made.
+
+## Where we've been
+
+We shared a few metrics in our Series A post. Since then we've grown a lot.
+
+### Database growth
+
+At Series A we had launched over 50,000 PostgreSQL databases (only on our hosted platform).
+
+![Series A databases](/images/blog/series-b/databases-2021.png)
+
+Since then, we've grown to over 150,000 PostgreSQL databases.
+
+![Series B databases](/images/blog/series-b/databases-2022.png)
+
+### **Developer Signups**
```

Comment:
@kiwicopple is this heading intentionally bolded?

Reply:
nope!